CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
```

Comment:
[Style] Please match the header indentation with the other subject markers.  this should be one more deep, so use ``###`` instead of just two.

Reply:
This does not fall under the `Naming & Commenting Standards` key, which was the previous `##` header. Thus, it belongs under its own `##` key and not tacked onto the end.